### ISSUE TITLE
Show a list of checked-in members

### ISSUE THREAD

**Issue by 0x736E** [Tog-Hackerspace/Website#30]
See: https://github.com/Tog-Hackerspace/Mobile-App/issues/10
For information on the "check-in" feature on the mobile app.

It would be nice to have a widget or area show who is presently in the space (if they wish to make that information available)

**Comment by estuans** [comment]
Perhaps if they chose to have it enabled, using location alone we could say a member is at the space without having any user interaction.

Say if they've been geofenced to the immediate area around tog for longer than 10-15 minutes the check them in?

**Comment by 0x736E** [comment]
Yes, that was the idea - it being an option which can be selectively turned on "automatically check in" or similar.

Alternatively, some may wish to check in manually, or not at all.

We should discuss the actual functionality of this feature on its parent ticket: Tog-Hackerspace/Mobile-App#10
